### PR TITLE
blog: add april monthly-report.md

### PR DESCRIPTION
**Pull request by Yilialinn**
Changes: Add the April monthly report for the Apache APISIX website.

### CODE REVIEW

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+description: Our monthly Apache APISIX community report is your window into the project's monthly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/30/XvSXCOav_monthly-report-apr-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. For detailed information, please read the monthly report.
+<!--truncate-->
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'Many hands make light work' rings true in our way, made possible by the collective dedication of our community.
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]: team work makes the dream works? ^^

**Review by kayx23 — DISMISSED** [review]

File: blog/zh/blog/2024/04/30/monthly-report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+![贡献者名单](https://static.apiseven.com/uploads/2024/04/30/txD3ooma_contributor-listi-apr.png)
+
+![新晋贡献者](https://static.apiseven.com/uploads/2024/04/30/b01wMlfs_new-contributors-apr.png)
+
+## 近期亮点功能
+
+- [新增 Kubernetes 服务的转存数据接口](https://github.com/apache/apisix/pull/11111)（贡献者：[hanqingwu](https://github.com/hanqingwu))
+
+- [引入新属性 max_resp_body_bytes 和 max_req_body_bytes 以降低 CPU 使用率](https://github.com/apache/apisix/pull/11133)（贡献者：[shreemaan-abhishek](https://github.com/shreemaan-abhishek))
```
Comment (kayx23) [inline blog/zh/blog/2024/04/30/monthly-report.md]: Why does this not mention the kafka-logger plugin

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2024/04/30/txD3ooma_contributor-listi-apr.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2024/04/30/b01wMlfs_new-contributors-apr.png)
+
+## Recent Highlights Features
+
+- [Add discovery k8s dump data interface](https://github.com/apache/apisix/pull/11111)（Contributor: [hanqingwu](https://github.com/hanqingwu))
+
+- [Add max req/resp body size attributes in Kafka Logger plugin](https://github.com/apache/apisix/pull/11133)（Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek))
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]:
```suggestion
- [Add max request and response body size attributes in the `kafka-logger` plugin](https://github.com/apache/apisix/pull/11133)（Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek))
```

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Monthly Report (April 01 - April 30)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report is your window into the project's monthly developments. It is a tool to facilitate your seamless integration into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]:
```suggestion
description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provides a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
```

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+From 04.01 to 04.30, a total of 16 contributors made 43 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2024/04/30/txD3ooma_contributor-listi-apr.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2024/04/30/b01wMlfs_new-contributors-apr.png)
+
+## Recent Highlights Features
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]:
```suggestion
## Recent Feature Highlights
```

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+- [Add discovery k8s dump data interface](https://github.com/apache/apisix/pull/11111)（Contributor: [hanqingwu](https://github.com/hanqingwu))
+
+- [Add max req/resp body size attributes in Kafka Logger plugin](https://github.com/apache/apisix/pull/11133)（Contributor: [shreemaan-abhishek](https://github.com/shreemaan-abhishek))
+
+- [Autogenerate Admin API key if not passed](https://github.com/apache/apisix/pull/11080)（Contributor: [Revolyssup](https://github.com/Revolyssup))
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]:
```suggestion
- [Autogenerate the admin API key if not configured in the configuration file](https://github.com/apache/apisix/pull/11080)（Contributor: [Revolyssup](https://github.com/Revolyssup))
```

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+## Recent Blog Recommendations
+
+- [Release Apache APISIX 3.8.1](https://apisix.apache.org/blog/2024/04/29/release-apache-apisix-3.8.1/)
+
+- [Release Apache APISIX 3.9.1](https://apisix.apache.org/blog/2024/04/29/release-apache-apisix-3.9.1/)
+
+The official website and GitHub Issues of Apache APISIX have accumulated rich documentation tutorials and usage experiences. If you encounter any issues, you can refer to the documentation, search for keywords in Issues, or participate in discussions on Issues to share your ideas and practical experiences.
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]: `accumulate rich documentation tutorials` --> `provide a wealth of documentation of tutorials, real-world use cases ...`

File: blog/zh/blog/2024/04/30/monthly-report.md (hunk 2 above)
Comment (kayx23) [reply blog/zh/blog/2024/04/30/monthly-report.md]: and the English version does not mention `max_resp_body_bytes` and `max_req_body_bytes`?

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provides a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/30/XvSXCOav_monthly-report-apr-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. For detailed information, please read the monthly report.
+<!--truncate-->
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way, made possible by the collective dedication of our community.
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]:
```suggestion
From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way and was made possible by the collective effort of our community.
```

File: blog/zh/blog/2024/04/30/monthly-report.md (hunk 2 above)
Comment (Yilialinn) [reply blog/zh/blog/2024/04/30/monthly-report.md]: Fixed

File: blog/zh/blog/2024/04/30/monthly-report.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+---
+title: "社区月报 (04.01 - 04.30)"
+keywords: ["Apache APISIX", "API 网关", "社区月报", "贡献者"]
+description: Apache APISIX 社区的月报旨在帮助社区成员更全面地了解社区的最新动态，方便大家参与到 Apache APISIX 社区中来。
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/30/0133BTQL_monthly-report-apr-cn.png
+---
+> 最近，我们新增并改进了 Apache APISIX 的部分功能，包含新增 Kubernetes 服务的转存数据接口、引入新属性 max_resp_body_bytes 和 max_req_body_bytes 以降低 CPU 使用率和自动生成 Admin API 密钥，以删除敏感 API 密钥的硬编码等。有关更多功能新亮点，请阅读本期月报。
```
Comment (kayx23) [inline blog/zh/blog/2024/04/30/monthly-report.md]:
```suggestion
> 最近，我们新增并改进了 Apache APISIX 的部分功能，包含新增 Kubernetes 服务的转存数据接口、在 `kafka-logger` 插件中引入新属性 `max_resp_body_bytes` 和 `max_req_body_bytes` 以降低 CPU 使用率，和自动生成 Admin API 密钥以删除硬编码的 Admin API 密钥等。有关更多功能新亮点，请阅读本期月报。
```

Comment (kayx23) [reply blog/zh/blog/2024/04/30/monthly-report.md]: `硬编码` not sure of the translation. I know it means hard-coded.

File: blog/en/blog/2024/04/30/monthly-report.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
+---
+title: "Monthly Report (April 01 - April 30)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/04/30/XvSXCOav_monthly-report-apr-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. For detailed information, please read the monthly report.
```
Comment (kayx23) [inline blog/en/blog/2024/04/30/monthly-report.md]: why's this different from

image

File: blog/zh/blog/2024/04/30/monthly-report.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+![新晋贡献者](https://static.apiseven.com/uploads/2024/04/30/b01wMlfs_new-contributors-apr.png)
+
+## 近期亮点功能
+
+- [新增 Kubernetes 服务的转存数据接口](https://github.com/apache/apisix/pull/11111)（贡献者：[hanqingwu](https://github.com/hanqingwu))
+
+- [在 `kafka-logger` 插件中引入新属性 `max_resp_body_bytes` 和 `max_req_body_bytes` 以降低 CPU 使用率](https://github.com/apache/apisix/pull/11133)（贡献者：[shreemaan-abhishek](https://github.com/shreemaan-abhishek))
+
+- [自动生成 Admin API 密钥，以删除敏感 API 密钥的硬编码](https://github.com/apache/apisix/pull/11080)（贡献者：[Revolyssup](https://github.com/Revolyssup))
```
Comment (kayx23) [inline blog/zh/blog/2024/04/30/monthly-report.md]:
```suggestion
- [自动生成 Admin API 密钥，以删除硬编码的 API 密钥](https://github.com/apache/apisix/pull/11080)（贡献者：[Revolyssup](https://github.com/Revolyssup))
```

Again not sure on the translation of `硬编码`. I'm assuming it's an adjective ~= hard-coded.

File: blog/zh/blog/2024/04/30/monthly-report.md (hunk 9 above)
Comment (Yilialinn) [reply blog/zh/blog/2024/04/30/monthly-report.md]: > `硬编码` not sure of the translation. I know it means hard-coded.

I think yes: https://zh.wikipedia.org/wiki/%E5%AF%AB%E6%AD%BB

**Review by kayx23 — DISMISSED** [review]

**Review by kayx23 — APPROVED** [review]

**Review by shreemaan-abhishek — APPROVED** [review]

**Pull request merged**